src/lighteval/metrics/imports/bert_scorer.py

```diff
@@ -383,7 +383,6 @@ def __init__(
 
         self._baseline_vals = None
         self.baseline_path = baseline_path
-        self.use_custom_baseline = self.baseline_path is not None
```

Comment:
why did you remove this ?

Reply:
Sorry for the confusion, this PR builds on top of https://github.com/huggingface/lighteval/pull/311 which removes this unused variable.